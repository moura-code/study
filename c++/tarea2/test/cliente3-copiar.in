crearCliente 31 Casimira Gomez 29
copiarCliente
liberarCliente
Fin
