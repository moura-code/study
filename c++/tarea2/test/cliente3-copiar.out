Bienvenido al programa principal de Mercado FINGer. Por favor ingrese su comando:
1> Cliente creado de forma exitosa.
2> Cliente copiado. Datos de la copia:
Cliente Casimira Gomez
Id: 31
Edad: 29
3> Cliente liberado con exito.
4> Fin.
